src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1161,7 +1161,7 @@ public function addPaymentAction(int $orderId, Request $request): RedirectRespon
                             $orderId,
                             $data['date'],
                             $data['payment_method'],
-                            $data['amount'],
+                            str_replace(',', '.', $data['amount']),
```

Comment:
The comma can have different meanings depending in the context
https://docs.oracle.com/cd/E19455-01/806-0169/overview-9/index.html
replacing it like this is kinda ... rough

Reply:
Maybe a DataTransformer in the form input which takes the context currency
Must be tried to know if it's possible

Reply:
A discussion is in progress about this subject. I will let @PrestaShop/product-team give you the fix expected/planned